docs/tutorial.rst

```diff
@@ -0,0 +1,127 @@
+========
+Tutorial
+========
+
+Welcome to the Dask cassandra loader tutorial. This tutorial demonstrates the basics of using
+Dask cassandra loader using either a local or a remote Cassandra database.
+
+To install Dask cassandra loader, use
+
+.. code-block:: bash
+
+  pip install dask_cassandra_loader
+
+If you're using dask cassandra loader in a program, you will probably want to use a
+virtualenv and install Cerulean into that, together with your other
+dependencies.
+
+Setup
+=====
+
+The tutorial requires the creation of a keyspace in an existent Cassandra cluster. For this
+tutorial it is used the keyspace called **tutorial**. In this example it is assume the local
+client cqlsh is installed and configured accordingly.
+
+.. code-block:: bash
+
+    cqlsh -e "create keyspace tutorial with replication = {'class': 'SimpleStrategy', 'replication_factor': 1};"
+
+Once the keyspace is created the user needs to create a table and load it. To do that the
+user needs to run the **.cql** file under the **dask-cassandra-loader/docs** directory as follow:
+
+.. code-block:: bash
+
+    cqlsh --keyspace=tutorial -f tutorial.cql
+
+Once the table is loaded, the user will have a table called **tab1** with the following schema: 
+
+.. code-block:: sql
+
+  create table tab1(id int, year int, month int, day int, timest timestamp, lat float, lon float, PRIMARY KEY((id, year, month)));
+  
+The loaded data has two partitions due two distinct months.
+
+
+Dask cassandra loader
+=====================
+
+The first step to load a table from Cassandra into a Dask data-frame is to create :class:`dask_cassandra_loader.loader.Loader`.
+To do that the user should do the following:
+
+.. code-block:: python
+
+  from dask_cassandra_loader import Loader
+  
+  dask_cassandra_loader = Loader()
+
+
+Connect to Cassandra
+--------------------
+
+With the loader the user is then able to set a connection to an existent Cassandra cluster.
+In this example we assume the user is connecting to local cluster using the default credentials.
+
+.. code-block:: python
+
+  keyspace = 'tutorial'
+  clusters = ['127.0.0.1']
```

Comment:
`clusters` should be changed to `cluster` as it is a single cluster consisting out of one or more servers.

Reply:
Solved in df5c2e7